src/map/packets_struct.h

```diff
@@ -290,13 +290,24 @@ enum packet_headers {
 	rouletteinfoackType = 0xa1c,
 	roulettgenerateackType = 0xa20,
 	roulettercvitemackType = 0xa22,
-#if 0 // Unknown
+#if PACKETVER >= 20150513 // Not confirmed
```

Comment:
If this can help.

0x9f8 from known data probably can works at 2014-03-26 or newer. And really works on 2015-09-18 and newer.

Reply:
zero 1206.exe
For the task will not be able to display properly